config/initializers/application_insights.rb

```diff
@@ -0,0 +1,11 @@
+unless Rails.env.test?
```

Comment:
Would this be better as `if Rails.env.production?` given that the Azure environments will run in that environment? Or is there is a specific reason we'd want to try and load application insights in someone's local development environment?

Reply:
@pezholio removed the check entirely. I assume that was discussed.

Reply:
Yep, I'm happy from my end. might want to check in with @pezholio who's working with Craig today.